openapi/components/schemas/Plans/FlexiblePlan.yaml

```
@@ -1,15 +1,36 @@
 type: object
 allOf:
   - $ref: ../Common/CommonPlan.yaml
-  - properties:
-      id:
-        readOnly: false
-        description: The ID of plan to modify with given settings.
-        allOf:
-          - $ref: ../ResourceId.yaml
-  - required:
-    - id
-    - name
-    - currency
-    - productId
-    - pricing
+properties:
+  id:
+    readOnly: false
+    description: The ID of plan to modify with given settings.
```

nictpick: suggestion to make description more concise

```suggestion
    description: ID of a plan you wish to modify.
```